src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -228,6 +237,9 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
         DispatchMessage(&msg);
     }
     desktopSource.Close();
+
+    hwmdLock.lock();
+    hwndEditKeyboardNativeWindow = nullptr;
```

Comment:
Can we also add a statement for setting `hWndXamlIslandEditKeyboardWindow` to `nullptr` here?

Reply:
Added.